src/platform/forms-system/test/js/review/ReadOnlyArrayField.unit.spec.jsx

```diff
@@ -48,5 +48,6 @@ describe('Schemaform review: ReadOnlyArrayField', () => {
     expect(tree.subTree('.schemaform-array-readonly-header').text()).to.contain(
       'Item',
     );
+    expect(tree.everySubTree('.review').length).to.equal(1);
```

Comment:
This is using SkinDeep testing, so I didn't check the `dl > div > dd` or `dl > div > dt` structure specifically